### ISSUE TITLE
keybind profiles

### ISSUE THREAD

**Issue by matcool** [geode-sdk/CustomKeybinds#51]
each profile has the binds for all keys, and you can switch between them and create new ones etc, would be nice for altering your keybinds to play one level, without having to then manually undo the changes

**Comment by Aktimoose** [comment]
Agreed, this would be particularly useful for a lot of platformer levels (bump)
